### PR TITLE
add api/reviews to the project

### PR DESCRIPTION
**Pull request by Mycroftd**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -36,3 +36,26 @@ describe("/api/categories", () => {
...
+            body.reviews.forEach((reviews) => {
+                expect(reviews).toMatchObject({
+                    review_id: expect.any(Number),
+                    title: expect.any(String),
+                    designer: expect.any(String),
+                    review_img_url: expect.any(String),
+                    review_body: expect.any(String),
+                    category: expect.any(String),
+                    created_at:expect.any(String),
+                    votes: expect.any(Number),
```
Comment (johnhutcheon) [inline __tests__/app.test.js:56]: This is missing the 'owner' and 'comment_count' properties as specified on Trello ticket 4. Could you please add these?

File: models/games.model.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
- } 
\ No newline at end of file
+exports.selectAllCategories = () => {
+  return db.query("SELECT * FROM categories;").then((categories) => {
+    return categories.rows;
+  });
+};
+
+exports.selectAllReviews = () => {
+  return db.query("SELECT * FROM reviews ORDER BY created_at DESC;").then((reviews) => {
```
Comment (johnhutcheon) [inline models/games.model.js:10]: This needs to take into account the 'comment_count' property which is specified on the ticket.

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hi David. It looks like you've just missed the 'comment_count' property which is required for this ticket, if you could look over the Trello board for this one again. If you could resubmit with the changes in the comments please, that would be great 👍

File: controllers/games.controller.js [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
\ No newline at end of file
+exports.getAllCategories = (req, res, next) => {
+  selectAllCategories().then((categories) => {
+    res.status(200).send({ categories });
+  });
+};
+
+exports.getAllReviews = (req,res, next) => {
+  selectAllReviews().then((reviews) => {
+    console.log(reviews);
```
Comment (MRombi) [inline controllers/games.controller.js:14]: We should delete all the console.logs 🙂

**Review by MRombi — APPROVED** [review]
Good job David, delete that console log and you are good to merge 😄

**Pull request merged**